static/canvas.js

```diff
@@ -23,7 +23,7 @@ class Canvas {
         this.selectionColor = '#CC0000';                   // Border color of selected boxes.
         this.selectionWidth = 2;                           // Border fillSize of selected boxes.
         this.selectionCorner = 5;                          // Corner fillSize of selected boxes.
-        this.interval = 30;                                // Frequency to be redrawn.
+        this.interval = 40;                                // Frequency to be redrawn.
```

Comment:
Still playing around with this. The only problem of too much computation occurs with the play and draw thing (which I'm thinking that I may have to get rid of). This is the only instance when this.interval needs to be around 50+ to handle, which even then could break. I was reading on the difference between setInterval and setTimeout, however, at the moment, setTimeout  isn't working any better. Will do more research later.